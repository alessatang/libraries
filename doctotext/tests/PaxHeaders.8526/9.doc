30 atime=1389151479.183862518
30 ctime=1389151479.183862518
