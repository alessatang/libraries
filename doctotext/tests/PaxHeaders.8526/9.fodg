30 mtime=1384978086.741497438
30 atime=1389151479.184862514
30 ctime=1389151479.184862514
