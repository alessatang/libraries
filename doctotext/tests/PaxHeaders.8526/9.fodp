30 mtime=1384978086.744497439
29 atime=1389151479.55486097
29 ctime=1389151479.55486097
