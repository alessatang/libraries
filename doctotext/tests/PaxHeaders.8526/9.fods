30 mtime=1384978086.745497439
30 atime=1389151479.555860966
30 ctime=1389151479.555860966
