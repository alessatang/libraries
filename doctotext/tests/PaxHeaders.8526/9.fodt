30 mtime=1384978086.746497439
30 atime=1389151479.555860966
30 ctime=1389151479.555860966
