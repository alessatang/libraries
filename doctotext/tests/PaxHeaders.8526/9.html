30 atime=1389151479.555860966
30 ctime=1389151479.555860966
