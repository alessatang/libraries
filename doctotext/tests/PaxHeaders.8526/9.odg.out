30 mtime=1379553523.281588311
30 atime=1389151480.367857573
30 ctime=1389151480.367857573
